### PR TITLE
Added post request to Applications 

### PR DESCRIPTION
**Pull request by Anshul-Birla**

### CODE REVIEW

File: backend/config/config.js [hunk 1]
```diff
@@ -1 +1,11 @@
-// TODO implement config.js using dotenv to hide credentials
+require('dotenv').config();
+
+// input parameters based on env
+const dev = {
+    app: {
+        port: process.env.PORT || 9000
+    },
+    uri: process.env.MONGO_URI
```
Comment (jayrc7) [inline backend/config/config.js:8]: change to db_uri

File: backend/db/db_models.js [hunk 2]
```diff
@@ -1,12 +1,36 @@
...
 client.connect(err => {
 	const userCollection = client.db("sample").collection("users");
 	userCollection.find({}).toArray(function (err, docs) {
 		if (err) console.log("error...");
 		else console.log(docs);
 	});
 });
+*/
+
+function addTest(name, response){
```
Comment (jayrc7) [inline backend/db/db_models.js:18]: function needs a better name maybe submitApplication()?

File: backend/db/db_models.js [hunk 3]
```diff
@@ -1,12 +1,36 @@
...
+	client.connect(err => {
+		if (err) throw err;
+		var dbo = client.db("tse_recruitment_db");
+		var myobj = { Name: name, Year: 2020, Response: response, Acceped:false };
```
Comment (jayrc7) [inline backend/db/db_models.js:22]: 2020 shouldn't be hardcoded, assume that you will be given the year as well

Comment (Anshul-Birla) [reply backend/db/db_models.js]: It says to default to current year in the team doc, so i just used Date().getFullYear() in the recent changes

File: backend/db/db_models.js (hunk 2 above)
Comment (Anshul-Birla) [reply backend/db/db_models.js]: my bad. fixed now

**Pull request merged**